00-HTML-CSS-basics/Ex1/form.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"/>
+  <meta name="author" content="Jacobo Andrés Dominguez"/>
+  <meta name="description" content="Exercise 1 | Doctypes and Metatags"/>
+  <meta name="keywords" content="Globant, Bootcamp, HTML5"/>
+  <meta name="viewport" content="width=device-width,initial-scale=1"/>
+  <title>Form</title>
+</head>
+<body>
+  <header>
+    <h2>Sign up form</h2>
+  </header>
+  <form>
+    <label for="firstName">First name</label>
+    <br />
+    <input type="text" name="firstName" placeholder="Type your first name..." required="true"/>
```

Comment:
Is not needed to set `required="true"` to set only `required` will set it as true by default